nixos/modules/misc/documentation.nix

```diff
@@ -113,36 +109,27 @@ in
         type = types.bool;
         default = true;
         description = ''
-          Whether to install manual pages and the <command>man</command> command.
-          This also includes "man" outputs.
+          Whether to install manual pages.
+          This also includes <literal>man</literal> outputs.
         '';
       };
 
       man.generateCaches = mkOption {
         type = types.bool;
         default = false;
         description = ''
-          Whether to generate the manual page index caches using
-          <literal>mandb(8)</literal>. This allows searching for a page or
-          keyword using utilities like <literal>apropos(1)</literal>.
-        '';
-      };
-
-      man.manualPages = mkOption {
-        type = types.path;
```

Comment:
Does this mean all configs which use `man.manualPackages` will break after this change?
If so, why not leave it? There’s no sense in breaking backwards compat for users. Maybe add a warning.

Reply:
Nope, `mkRenamedOptionModule` should mean that this only becomes a warning and is forwarded to the new place gracefully. Changelog entry is mostly for completeness' sake.